SConstruct

```diff
@@ -97,9 +89,76 @@ LSSTComCam_injected = [
     )
     for suffix, pipeline_post in injected_pipelines_LSSTComCam.items()
 ]
+
+# diffim injection pipeline creation
+subset_names = "injected_diffim_analysis,injected_stage4-measure-variability"
+subset_description = "Analysis tasks for source injection for diffim catalogs"
```

Comment:
This isn't currently being used. That said, considering the PR comment in the `source_injection` repo, this may be switched up substantially following an API change.